README.md

```diff
@@ -57,6 +57,37 @@ The notification processing service queries the records that have been received
 
 If you need to setup your cronjob in Magento <a href="http://devdocs.magento.com/guides/v2.0/config-guide/cli/config-cli-subcommands-cron.html" target="_blank">this is described here</a>
 
+##Verified payment methods
+
+ * Card payment methods
+    * Card payment method (non 3D, 3DS1 and 3DS2)
+    * One click payment methods (using billing agreement or Vault)
+    * Recurring payment methods (using billing agreement or Vault) 
+   
+
+* Local payment methods
+   * Afterpay
+   * Apple Pay
+   * Bancontact
+   * Bantransfer IBAN
+   * Doku
+   * Google Pay
+   * iDeal
+   * Klarna
+   * Oney
+   * Zip
+   * Multibanco
+   * Konbini
+   * Swish
+   * MBway
+   * Twint
+   * Paypal
+   * SEPA Direct Debit
+   * PIX
+   * Ratepay
+   * Oxxo
+   * Econtexts
+
```

Comment:
This is not an extensive list of all the supported payment methods but only the ones that we have already verified. In case you would like to see other payment methods in the list as well please contact us to verify those as well.